tests/Aspire.Hosting.PostgreSQL.Tests/PostgresFunctionalTests.cs

```diff
@@ -218,7 +218,14 @@ public async Task VerifyWithPgWeb()
 
         IResourceBuilder<PgWebContainerResource>? pgWebBuilder = null;
         var dbName = "postgres";
-        var pg = builder.AddPostgres("pg1").WithPgWeb(c => pgWebBuilder = c).AddDatabase(dbName);
+        var pg = builder.AddPostgres("pg1");
+        var db = pg.AddDatabase(dbName);
+        pg.WithPgWeb(c =>
+        {
+            c.WaitFor(db);
```

Comment:
What creates the database? We have an issue today where the database health check won't get go healthy until the database exists. Since the apphost doesn't create the database, it means that it'll hang forever.

Reply:
In this test, we use `postgres` database which is the default database in the Postgres container and by default the `postgres` database exists.

Reply:
Why not just wait on the server?

Reply:
```csharp

        var httpContent = new MultipartFormDataContent
        {
            { new StringContent(dbName), "bookmark_id" }
        };

        client.DefaultRequestHeaders.Add("x-session-id", Guid.NewGuid().ToString());

        var response = await client.PostAsync("/api/connect", httpContent);
```

Beacause `api/connect` tries connect to `postgres` db.